HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -63,6 +63,29 @@ def customiseFor20429(process):
         del producer.GBRForestFileName
     return process
 
+
+# 
+# Threshold 1-sigma backported from 10_0_X cmssw release from 2018 to perform tests on 2017 data/MC
+#
+
+def customiseForEcalTest(process):
```

Comment:
This PR is proposed as a part of data rereco request in 94X. Is the change in the HLT configuration even needed in this context?

Reply:
@amassiro 
HLT is not re-run in data re-reco